saleor/payment/models.py

```diff
@@ -67,6 +67,15 @@ class TransactionItem(ModelWithMetadata):
         default=Decimal("0"),
     )
 
+    amount_pending_refund = MoneyField(
+        amount_field="pending_refund_value", currency_field="currency"
+    )
+    pending_refund_value = models.DecimalField(
```

Comment:
Imo the naming here should follow the same pattern: pending_refund_amount, pending_refund_value.

Reply:
@maarcingebala I used the same pattern as we have for the rest of the fields:
```
amount_voided = MoneyField(amount_field="voided_value", currency_field="currency")
voided_value = models.DecimalField(
        max_digits=settings.DEFAULT_MAX_DIGITS,
        decimal_places=settings.DEFAULT_DECIMAL_PLACES,
        default=Decimal("0"),
    )
```

Reply:
I see, but don't you think it's a bad pattern? :p

Reply:
Honestly, I don't have an opinion, but I would prefer to have the same naming convention for all fields. If you think that the field names should be changed then IMHO we should re-name all fields in separate PR.

Reply:
I will prepare a separate PR for renaming all fields in the same time. I will do that after we will merge the changes related to transaction